deps/build.jl

```diff
@@ -13,8 +13,16 @@ module DefaultDeps
     if !isdefined(@__MODULE__, :ROOTENV)
         const ROOTENV = joinpath(Main.condadir, MINICONDA_VERSION)
     end
+
+    USE_MINIFORGE_DEFAULT = true
+    if Sys.ARCH in [:x86, :i686]:
+        USE_MINIFORGE_DEFAULT = false
+        @warn """Using defaults channel as conda-forge is not supported on this platform.
+        Using defaults packages requires a license for some use-cases.
```

Comment:
```suggestion
        @warn """The free/open-source Miniforge (i.e. the conda-forge defaults channel) does not support this platform.
Using the Anaconda channel instead, which is free for non-commercial use but otherwise may require a license.
```